src/main/java/uk/gov/hmcts/reform/idam/client/OAuth2Configuration.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.idam.client;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Getter
+@Component
+public class OAuth2Configuration {
```

Comment:
would be tempted to just create this as a kotlin class then the getters are there for you anyway.